repos/exawind/packages/hypre/package.py

```diff
@@ -7,6 +7,8 @@ class Hypre(bHypre):
     phases = ['autoreconf', 'distclean', 'configure', 'clean', 'build', 'install']
 
     variant('gpu-aware-mpi', default=False, description='Use gpu-aware mpi')
+    variant('rocblas', default=False, description='use rocblas')
+    variant('cublas', default=False, description='use cublas')
```

Comment:
```suggestion
    variant('cublas', default=False, description='use cublas')
    conflict('+cublas', when='~cuda', msg='cublas requires cuda to be enabled')
    conflict('+rocblas', when='~rocm', msg='rocblas requires rocm to be enabled')
```
We should add some safety rails to this.

Reply:
I like the safety checks

Reply:
whoops it's `conflicts` with an `s` my bad.